tools/Dockerfile.build.ppc64le

```diff
@@ -0,0 +1,51 @@
+# docker build --build-arg UID=$(id -u) -t nvidia-docker -f Dockerfile.build.ppc64le .
+
+
+FROM ppc64le/golang:1.6.3
+
+ENV NVIDIA_GPGKEY_SUM bd841d59a27a406e513db7d405550894188a4c1cd96bf8aa4f82f1b39e0b5c1c
+ENV NVIDIA_GPGKEY_FPR 889bee522da690103c4b085ed88c3d385c37d3be
+ENV NVIDIA_GDK_SUM 1e32e58f69fe29ee67b845233e7aa9347f37994463252bccbc8bfc8a7104ab5a
+
+RUN curl http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1404/ppc64el/cuda-repo-ubuntu1404_7.5-18_ppc64el.deb -o cudarepo.deb && dpkg -i cudarepo.deb
```

Comment:
Please make it consistent with the rest of the `nvidia-docker` project regarding `curl` flags (`-fsSL`), you should also line wrap for the `dpkg` command.